src/MagnumPlugins/TinyGltfImporter/Test/TinyGltfImporterTest.cpp

```diff
@@ -2517,6 +2519,33 @@ void TinyGltfImporterTest::materialTexCoordFlip() {
     }), TestSuite::Compare::Container);
 }
 
+void TinyGltfImporterTest::materialTextureCoordinateSets() {
+    Containers::Pointer<AbstractImporter> importer = _manager.instantiate("TinyGltfImporter");
+    importer->configuration().setValue("allowMaterialTextureCoordinateSets", true);
```

Comment:
I would make two variants of this test, the other testing the default behavior, that the import fails for each of the materials.